README.md

```diff
@@ -1,30 +1,52 @@
-# Sourcegraph extension for Phabricator
+# Sourcegraph's Phabricator integration
 
 Get code intelligence on Phabricator with Sourcegraph.
 
 Check out our [docs site](https://about.sourcegraph.com/docs/features/phabricator-extension) for more information.
 
 ## [Installation](https://https://about.sourcegraph.com/docs/features/phabricator-extension#option-a-single-installation-script)
 
-1. To install Sourcegraph on your Phabricator instance run `./install.sh <path/to/phabricator/root> <sourcegraph url>` inside your Phabricator instance.
-2. Restart Phabricator instance.
+1. Clone this repository into a sourcegraph folder located in your Phabricator extension directory.
 
-Requires PHP 5.5 or newer and `php-tokenizer` (a default PHP extension).
+```
+git clone -b v1.0 git@github.com:sourcegraph/phabricator-extension-install.git path/to/phabricator/src/extensions/sourcegraph
```

Comment:
Is `v1.0` a release branch that will have future commits for the `v1.0.x` series of releases? If so, I would recommend including "release" in the name so it doesn't get confused with tags.

Reply:
Also, it is better to use an `https:` clone URL for because this will be run on servers, many of whom have never seen github.com before, so the user would be prompted to accept the SSH host key each time.